app/src/main/java/com/weatherxm/ui/photoverification/gallery/PhotoGalleryActivity.kt

```diff
@@ -234,6 +260,11 @@ class PhotoGalleryActivity : BaseActivity() {
                 )
                 getCameraPermissions()
             }
+
+            binding.galleryBtn.setOnClickListener {
+                // TODO: track event
+                openPhotoPicker()
```

Comment:
No need I think to create a different function, just call directly `navigator.openPhotoPicker(photoPickerLauncher)` as it's the only place we call that.